plugins/vendor/zoom/ZoomDropFolderPlugin/lib/api/KalturaZoomDropFolder.php

```diff
@@ -121,6 +122,22 @@ public function doFromObject($sourceObject, KalturaDetachedResponseProfile $resp
 		
 	}
 	
+	public function toObject($dbObject = null, $skip = array())
+	{
+		/* @var ZoomVendorIntegration $vendorIntegration */
+		$vendorIntegration = VendorIntegrationPeer::retrieveByPK($dbObject->getZoomVendorIntegrationId());
+		$vendorIntegration->setZoomAccountDescription($dbObject->getDescription());
+		$vendorIntegration->save();
+		
+		if (!$dbObject)
```

Comment:
is it really possible that we will come here without dbobject, if not avoid creating new one - and throw